go/activities.go

```diff
@@ -0,0 +1,14 @@
+package loyalty
+
+import (
+	"context"
+	"go.temporal.io/sdk/activity"
+)
+
+type Activities struct{}
+
+func (a *Activities) SendEmail(ctx context.Context, body string) (err error) {
```

Comment:
```suggestion
func (*Activities) SendEmail(ctx context.Context, body string) error {
```

No need to name variables you don't use